README.md

```diff
@@ -1,4 +1,16 @@
 # UltracoolTypingKit
 Home for qualitative spectral typing GUI that follows the Cruz Method
 
+
 ### Instructions on usage coming soon!
+
```

Comment:
Along with instructions I think it would be useful to provide a short explanation of the different spectral types. As a person who has no experience with these types they all look the same to me.